admin/src/components/DeletedUserFormular.spec.js

```diff
@@ -49,198 +49,198 @@ describe('DeletedUserFormular', () => {
     it('has a DIV element with the class.delete-user-formular', () => {
       expect(wrapper.find('.deleted-user-formular').exists()).toBeTruthy()
     })
-  })
 
-  describe('delete self', () => {
-    beforeEach(() => {
-      wrapper.setProps({
-        item: {
-          userId: 0,
-        },
+    describe('delete self', () => {
+      beforeEach(() => {
+        wrapper.setProps({
+          item: {
+            userId: 0,
+          },
+        })
```

Comment:
I included all following tests into the `mount` describe block, because only then the `wrapper` variable is set always for save.

Why this test file was always successful I have no answer.
As I have it overtaken it always failed, because `wrapper` was undefined in the describe blocks after `mount`.

Reply:
@ogerly confirmed this 🙏🏼